src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -822,7 +822,8 @@ class HaAutomationPicker extends SubscribeMixin(LitElement) {
         </ha-fab>
       </hass-tabs-subpage-data-table>
       <ha-menu id="overflow-menu" positioning="fixed">
-        <ha-menu-item @click=${this._showInfo}>
+        <ha-menu-item .action=${this._showInfo}
```

Comment:
We should define `action` as a property on `ha-menu-item` in addition to moving the listener to `ha-menu`.